From 25178d5203e1e8d30e7197790e90ad61c7a2c470 Mon Sep 17 00:00:00 2001
From: Neta Zur <neta@marvell.com>
Date: Mon, 18 Jan 2016 17:31:02 +0200
Subject: [PATCH 0703/1240] config: a8k: align the defconfig files to
 savedefconfig

Update the defconfig files according to the output of the
'make savedefconfig' command

Change-Id: Ie05c1f8fddfbbc367de03baf97df1eeb845d05e4
Signed-off-by: Neta Zur <neta@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/26858
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 configs/mvebu_apn806_defconfig        |  6 +++---
 configs/mvebu_apn806_dop_defconfig    | 12 +++++-------
 configs/mvebu_apn806_pd_defconfig     |  4 ++--
 configs/mvebu_armada70x0_defconfig    |  7 +++----
 configs/mvebu_armada70x0_pd_defconfig |  4 ++--
 5 files changed, 15 insertions(+), 18 deletions(-)

diff --git a/configs/mvebu_apn806_defconfig b/configs/mvebu_apn806_defconfig
index 232ff20..c2b023f 100644
--- a/configs/mvebu_apn806_defconfig
+++ b/configs/mvebu_apn806_defconfig
@@ -2,7 +2,10 @@ CONFIG_SPL=y
 CONFIG_SYS_EXTRA_OPTIONS="ARM64"
 +S:CONFIG_ARM=y
 +S:CONFIG_TARGET_ARMADA_8K=y
+CONFIG_MVEBU_CCU=y
+CONFIG_MVEBU_RFU=y
 CONFIG_MVEBU_MPP_BUS=y
++S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
 +S:CONFIG_DEVEL_BOARD=y
 CONFIG_CMD_BOOTD=y
 CONFIG_CMD_RUN=y
@@ -25,8 +28,6 @@ CONFIG_CMD_MVEBU_TSEN=y
 +S:CONFIG_OF_EMBED=y
 +S:CONFIG_DEFAULT_DEVICE_TREE="apn-806-db"
 CONFIG_MVEBU_PCI=y
-CONFIG_MVEBU_CCU=y
-CONFIG_MVEBU_RFU=y
 CONFIG_ENC28J60=y
 +S:CONFIG_SYS_NS16550=y
 +S:CONFIG_MVEBU_I2C=y
@@ -36,4 +37,3 @@ CONFIG_MVEBU_PINCTL=y
 CONFIG_MVEBU_THERMAL_SENSOR=y
 CONFIG_MVEBU_THERMAL_SENSOR_28NM_V2=y
 +S:CONFIG_MVEBU_MK6_SUPPORT=y
-+S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
diff --git a/configs/mvebu_apn806_dop_defconfig b/configs/mvebu_apn806_dop_defconfig
index d029385..7631333 100644
--- a/configs/mvebu_apn806_dop_defconfig
+++ b/configs/mvebu_apn806_dop_defconfig
@@ -2,20 +2,18 @@ CONFIG_SPL=y
 CONFIG_SYS_EXTRA_OPTIONS="ARM64"
 +S:CONFIG_ARM=y
 +S:CONFIG_TARGET_ARMADA_8K=y
-# CONFIG_PALLADIUM is not set
-CONFIG_MVEBU_SPI=y
 CONFIG_MVEBU_MPP_BUS=y
-CONFIG_MVEBU_I2C=y
 +S:CONFIG_MVEBU_SPL_DDR_OVER_PCI_SUPPORT=y
++S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
 +S:CONFIG_MVEBU_SPL_SAR_DUMP=y
 +S:CONFIG_DEVEL_BOARD=y
++S:# CONFIG_MULTI_DT_FILE is not set
 CONFIG_CMD_BOOTD=y
 CONFIG_CMD_RUN=y
 CONFIG_CMD_EDITENV=y
 CONFIG_CMD_SAVEENV=y
 CONFIG_CMD_MEMORY=y
 CONFIG_CMD_MEMTEST=y
-CONFIG_CMD_I2C=y
 CONFIG_CMD_NET=y
 CONFIG_CMD_PING=y
 CONFIG_CMD_TIME=y
@@ -30,10 +28,10 @@ CONFIG_CMD_MVEBU_TSEN=y
 +S:CONFIG_DEFAULT_DEVICE_TREE="apn-806-db-dop"
 CONFIG_ENC28J60=y
 +S:CONFIG_SYS_NS16550=y
+CONFIG_MVEBU_I2C=y
+CONFIG_MVEBU_SPI=y
 CONFIG_MVEBU_PINCTL=y
 +S:CONFIG_MVEBU_COMPHY_SUPPORT=y
-+S:CONFIG_MVEBU_MK6_SUPPORT=y
 CONFIG_MVEBU_THERMAL_SENSOR=y
 CONFIG_MVEBU_THERMAL_SENSOR_28NM_V2=y
-+S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
-+S:# CONFIG_MULTI_DT_FILE is not set
++S:CONFIG_MVEBU_MK6_SUPPORT=y
diff --git a/configs/mvebu_apn806_pd_defconfig b/configs/mvebu_apn806_pd_defconfig
index 35b6125..3286932 100644
--- a/configs/mvebu_apn806_pd_defconfig
+++ b/configs/mvebu_apn806_pd_defconfig
@@ -6,7 +6,9 @@ CONFIG_PALLADIUM=y
 CONFIG_MVEBU_CCU=y
 CONFIG_MVEBU_RFU=y
 CONFIG_MVEBU_MPP_BUS=y
++S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
 +S:CONFIG_DEVEL_BOARD=y
++S:# CONFIG_MULTI_DT_FILE is not set
 CONFIG_CMD_BOOTD=y
 CONFIG_CMD_RUN=y
 CONFIG_CMD_EDITENV=y
@@ -24,5 +26,3 @@ CONFIG_CMD_MVEBU_MPP=y
 +S:CONFIG_SYS_NS16550=y
 CONFIG_MVEBU_PINCTL=y
 +S:# CONFIG_MVEBU_SPL_DDR_SUPPORT is not set
-+S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
-+S:# CONFIG_MULTI_DT_FILE is not set
diff --git a/configs/mvebu_armada70x0_defconfig b/configs/mvebu_armada70x0_defconfig
index ae953e7..5e565fd 100644
--- a/configs/mvebu_armada70x0_defconfig
+++ b/configs/mvebu_armada70x0_defconfig
@@ -2,9 +2,11 @@ CONFIG_SPL=y
 CONFIG_SYS_EXTRA_OPTIONS="ARM64"
 +S:CONFIG_ARM=y
 +S:CONFIG_TARGET_ARMADA_8K=y
+CONFIG_MVEBU_CCU=y
+CONFIG_MVEBU_RFU=y
 CONFIG_MVEBU_MPP_BUS=y
++S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
 +S:CONFIG_DEVEL_BOARD=y
-CONFIG_ARMADA_70X0_DB=y
 CONFIG_CMD_BOOTD=y
 CONFIG_CMD_RUN=y
 CONFIG_CMD_EDITENV=y
@@ -25,8 +27,6 @@ CONFIG_CMD_MVEBU_TSEN=y
 +S:CONFIG_OF_EMBED=y
 +S:CONFIG_DEFAULT_DEVICE_TREE="armada-7040-rz-db_router"
 CONFIG_MVEBU_PCI=y
-CONFIG_MVEBU_CCU=y
-CONFIG_MVEBU_RFU=y
 CONFIG_ENC28J60=y
 +S:CONFIG_SYS_NS16550=y
 +S:CONFIG_MVEBU_I2C=y
@@ -36,4 +36,3 @@ CONFIG_MVEBU_PINCTL=y
 CONFIG_MVEBU_THERMAL_SENSOR=y
 CONFIG_MVEBU_THERMAL_SENSOR_28NM_V2=y
 +S:CONFIG_MVEBU_MK6_SUPPORT=y
-+S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
diff --git a/configs/mvebu_armada70x0_pd_defconfig b/configs/mvebu_armada70x0_pd_defconfig
index 32c30f9..793cdf7 100644
--- a/configs/mvebu_armada70x0_pd_defconfig
+++ b/configs/mvebu_armada70x0_pd_defconfig
@@ -9,8 +9,10 @@ CONFIG_MVEBU_IOB=y
 CONFIG_MVEBU_RFU=y
 CONFIG_MVEBU_MPP_BUS=y
 # CONFIG_MVEBU_TOOLS_SUPPORT is not set
++S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
 +S:CONFIG_DEVEL_BOARD=y
 # CONFIG_MVEBU_SAR is not set
++S:# CONFIG_MULTI_DT_FILE is not set
 CONFIG_CMD_BOOTD=y
 CONFIG_CMD_RUN=y
 CONFIG_CMD_EDITENV=y
@@ -36,5 +38,3 @@ CONFIG_MVEBU_SPI=y
 CONFIG_MVEBU_PINCTL=y
 +S:CONFIG_MVEBU_COMPHY_SUPPORT=y
 +S:# CONFIG_MVEBU_SPL_DDR_SUPPORT is not set
-+S:CONFIG_MVEBU_ROM_SILENCE_FLAG_SUPPORT=y
-+S:# CONFIG_MULTI_DT_FILE is not set
-- 
1.9.1

